sites/docs/src/content/docs/tutorials/tests_and_test_data/components/02_commands_integration.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "2. nf-test Commands & Integration"
+subtitle: Essential commands and nf-core integration
+weight: 20
+---
+
+## Prerequisites
+
+Before running these commands, ensure you have:
+
+- nf-test installed (see [Installation Guide](./01_installation.md))
+- An nf-core pipeline or nf-core/modules repo set up
+- Access to test data (local or remote)
+
+## Understanding Testing Contexts
+
+There are **two main contexts** for running nf-test commands in the nf-core ecosystem:
+
+### 1. nf-core/modules Repository Context
+
+When working in the **nf-core/modules repository**, use `nf-core` tools commands:
+
+- Testing individual modules and subworkflows
+- Contributing to the shared nf-core modules library
+- Use `nf-core modules test` and `nf-core subworkflows test`
+
+### 2. Individual Pipeline Context
+
+When developing **individual nf-core pipelines**, use `nf-test` commands directly:
+
+- Testing complete pipeline workflows
+- Pipeline-specific test configurations
+- Use `nf-test test` commands
+
+---
+
+## nf-core/modules Repository Commands
+
+> **Note**: These commands only work within the nf-core/modules repository
+
+### Module Testing
+
+```bash
+# Test a specific module
+nf-core modules test bedtools/bamtobed --profile docker
+
+# Update module test snapshots
+nf-core modules test bedtools/bamtobed --profile docker --update
+
+# Test with verbose output
+nf-core modules test bedtools/bamtobed --profile docker --verbose
+```
+
+### Subworkflow Testing
+
+```bash
+# Test a subworkflow
+nf-core subworkflows test vcf_impute_glimpse --profile docker
+
+# Update subworkflow snapshots
+nf-core subworkflows test vcf_impute_glimpse --profile docker --update
+```
+
+---
+
+## Individual Pipeline Commands
+
+> **Note**: Use these commands when working on individual nf-core pipelines. You can use `nf-core/sarek`, `nf-core/methylseq`, `nf-core/rnaseq` as examples.
+
+### Basic Testing
+
+```bash
+# List all available tests
+nf-test list
+
+# Run all tests with profiles
```

Comment:
I would make sure to note this will run EVERYTHING (including modules etc, it's not just pipeline tests)